### PR TITLE
Fix trees_loaded variable's memory corruption.

### PR DESCRIPTION
**Pull request by NikhilBirgade**
Signed-off-by: Nikhil Kumar Birgade <nikhil.birgade@seagate.com>

### CODE REVIEW

File: btree/btree.c [hunk 1]
```diff
@@ -2332,8 +2332,8 @@ static void mem_tree_put(struct td *tree)
 		m0_rwlock_write_lock(&trees_lock);
 		M0_ASSERT(trees_loaded > 0);
 		i = tree - &trees[0];
-		array_offset = i / sizeof(trees_in_use[0]);
-		bit_offset_in_array = i % sizeof(trees_in_use[0]);
+		array_offset = i / (sizeof(trees_in_use[0]) * 8);
+		bit_offset_in_array = i % (sizeof(trees_in_use[0]) * 8);
```
Comment (RadhaGulhane13) [inline btree/btree.c:2336]: 8 is hardcoded can we replace it by defining cost variable and use it here

Comment (NikhilBirgade) [reply btree/btree.c:2336]: Multiplying by 8 to convert it to bits. Anyways, this code will be replaced after tree_get/put changes.

Comment (shashank-parulekar) [inline btree/btree.c:2336]: Good catch. This same change needs to be done in tree_get routine where the offset it calculated at line 2244 in this file.

Comment (RadhaGulhane13) [reply btree/btree.c:2336]: okay

**Review by shashank-parulekar — CHANGES_REQUESTED** [review]

**Review by RadhaGulhane13 — APPROVED** [review]

**Review by shashank-parulekar — APPROVED** [review]

File: btree/btree.c (hunk 1 above)
Comment (NikhilBirgade) [reply btree/btree.c:2336]: Oh yes. Thanks.

**Pull request merged**